src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -91,18 +91,35 @@ const FiltersCountView = View.extend({
   },
 });
 
+const ClearFiltersView = View.extend({
+  template: hbs`
+    <button class="filters-sidebar__clear-filters u-margin--r-16 js-clear-filters" {{#if isDisabled}}disabled{{/if}}>
+      {{ @intl.patients.sidebar.filters.filtersSidebarViews.clearFilters }}
+    </button>
+  `,
+  templateContext() {
+    return {
+      isDisabled: this.getOption('isDisabled'),
+    };
+  },
+});
```

Comment:
Moved the `Clear Filters` button to its own child view so I could render it whenever the filters count changed.